libraries/standard/mqtt/cbmc/proofs/MQTT_GetIncomingPacketTypeAndLength/cbmc-batch.yaml

```diff
@@ -0,0 +1,2 @@
+# This file marks this directory as containing a CBMC proof.
+# The contents of this file will be generated by continuous integration.
```

Comment:
Any idea why these files do not end with a newline?  I'm pretty sure they exist in the templates.

Reply:
They do not seem to end with a newline by default for me either.